# Topics that RRT listens to
pose_topic: "odom"
scan_topic: "scan"

# Topics that RRT publishes to
local_path_topic: "local_path"

#######
# Parameters for RRT
#######

# Max Iterations of RRT before giving up
max_rrt_iters: 2500

# Max Value for steer between two nodes
max_expansion_distance: 1.0 # in meters

# Dicretization along edge joining two nodes in RRT
collision_checking_points: 10

# RRT Local Planner Goal Tolerance
goal_tolerance: 0.1

# Lookahead Distance for tracking reference of local planner (RRT)
lookahead_distance: 5.0 # in meters

# Near search radius for a node in the RRT* tree
search_radius: 1.5

# Enable RRT*
enable_rrtstar: true

# Obstacle Inflation
inflation_radius: 3

